src/Perpetuum/Zones/Terrains/Materials/Plants/PlantHandler.cs

```diff
@@ -21,10 +21,15 @@ public class PlantHandler : IPlantHandler
         public delegate IPlantHandler Factory(IZone zone);
 
         private const int AREA_SIZE = 32;
+        // The number of 32x32 cubes across the zone max width of 2048x2048
+        private const int TIME_SCALING_BASE = 64;
+        // Number of seconds in 8 hours
+        private const int EIGHT_HOURS = 28800;
+        private const int PLANT_REGEN_BASE_DELAY = EIGHT_HOURS / TIME_SCALING_BASE;
 
         private readonly IZone _zone;
         private readonly TimeSpan _natureSleepAmount = TimeSpan.FromSeconds(7);
-        private IntervalTimer _plantsTimer = new IntervalTimer(TimeSpan.FromSeconds(7));
+        private IntervalTimer _plantsTimer = new IntervalTimer(TimeSpan.FromSeconds(PLANT_REGEN_BASE_DELAY));
```

Comment:
Hang on, you compute it for the 2048 assumption, and then call `AddTimerScaling` on the actual width later, so this work here has no effect in the end.

The `((TIME_SCALING_BASE / zoneWidth) * 7));` x7 bit is the part I'd like to avoid as that is even more unintuitive if you don't know what the secret is there

Let's try something like this instead:
```cs
private readonly TimeSpan FULL_PASS = TimeSpan.FromHours(8); // So we know a total time that we to divide by the number of cube/areas we need to process
...
var totalAreas = (_zone.Size.Width / AREA_SIZE) * (_zone.Size.Height / AREA_SIZE); // total number of 32x32s in the actual zone size
var timePerArea = FULL_PASS / totalAreas; // 8 hours divided by the number of areas
_plantsTimer = new IntervalTimer(timePerArea); //timePerArea should be a TimeSpan
```

Hope that clears up what I meant in the last review